src/Hive.Core/sia380/sia380.py

```diff
@@ -205,35 +206,43 @@ def main(room_properties, floor_area, T_e, setpoints_ub, setpoints_lb, surface_a
     # preAllocate arrays. its a bit faster than .append (https://levelup.gitconnected.com/faster-lists-in-python-4c4287502f0a)
     Q_i = [0.0] * months_per_year
     Q_s = [0.0] * months_per_year
-    Q_i_eta_g = [0.0] * months_per_year
-    Q_s_eta_g = [0.0] * months_per_year
-    Q_V = [0.0] * months_per_year
-    Q_T = [0.0] * months_per_year
-    QT_opaque = [0.0] * months_per_year
-    QT_transparent = [0.0] * months_per_year
+    Q_i_out = [0.0] * months_per_year
+    Q_s_out = [0.0] * months_per_year
+    Q_V_out = [0.0] * months_per_year
+    Q_T_out = [0.0] * months_per_year
+    QT_opaque_out = [0.0] * months_per_year
+    QT_transparent_out = [0.0] * months_per_year
 
     Q_Heat = [0.0] * months_per_year
     Q_Cool = [0.0] * months_per_year
     Q_Elec = [0.0] * months_per_year
 
-
     Phi_P_tot = Phi_P * floor_area
     Phi_L_tot = Phi_L * floor_area
     Phi_A_tot = Phi_A * floor_area
 
+    simulation_cases = 4
```

Comment:
how did you come up with this number? a comment here would be very helpful.

Reply:
e.g. what is it even used for?

Reply:
oh yeah it's not used anymore. I had it before to initialize a bunch of arrays